feature-libs/asm/customer-360/components/sections/asm-customer-promotion/asm-customer-promotion.component.ts

```diff
@@ -0,0 +1,82 @@
+/*
+ * SPDX-FileCopyrightText: 2023 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import {
+  ChangeDetectionStrategy,
+  Component,
+  OnDestroy,
+  OnInit,
+} from '@angular/core';
+import {
+  Customer360Facade,
+  Customer360PromotionList,
+} from '@spartacus/asm/customer-360/root';
+import {
+  BehaviorSubject,
+  Observable,
+  Subscription,
+  combineLatest,
+  of,
+} from 'rxjs';
+import { catchError, map } from 'rxjs/operators';
+import { Customer360SectionContext } from '../customer-360-section-context.model';
+import { PromotionEntry } from './asm-customer-promotion.model';
+
+@Component({
+  changeDetection: ChangeDetectionStrategy.OnPush,
+  selector: 'cx-asm-customer-promotion',
+  templateUrl: './asm-customer-promotion.component.html',
+})
+export class AsmCustomerPromotionComponent implements OnInit, OnDestroy {
+  showErrorAlert$ = new BehaviorSubject<boolean>(false);
+  showErrorAlertForApplyAction$ = new BehaviorSubject<boolean>(false);
+  entries$: Observable<Array<PromotionEntry>>;
+  subscription = new Subscription();
+
+  constructor(
+    protected context: Customer360SectionContext<Customer360PromotionList>,
+    protected customer360Facade: Customer360Facade
```

Comment:
seems `Customer360Facade` is not used in promotion section? Let's remove it directly.